azavea/summer-of-maps-website
# Change client name (from DVAEYC to First Up)

## Overview
A former SoM client (DVAEYC) has changed their company name and website URL. Since theirs is prominently displayed as a featured project, and we occasionally share their shortlist page with prospects, we want to update this in several places.

@dmcglone and I discussed offline, and agreed that it might be best to change instances of the name to **First Up (formerly DVAEYC)**.

### Demo
<img width="1792" alt="Screen Shot 2019-06-20 at 12 10 32 PM" src="https://user-images.githubusercontent.com/5672295/59864921-cf3e9880-9355-11e9-9c4c-4ebe71ec3070.png">
<img width="432" alt="Screen Shot 2019-06-20 at 12 11 24 PM" src="https://user-images.githubusercontent.com/5672295/59864922-cf3e9880-9355-11e9-8026-0868002c77e6.png">
<img width="1792" alt="Screen Shot 2019-06-20 at 12 11 30 PM" src="https://user-images.githubusercontent.com/5672295/59864923-cf3e9880-9355-11e9-9189-2e75d2cf8b66.png">
<img width="799" alt="Screen Shot 2019-06-20 at 12 28 22 PM" src="https://user-images.githubusercontent.com/5672295/59865411-e9c54180-9356-11e9-8470-ff3fb56e5fab.png">


### Notes
* Since images of the final project include the original client name, we are including some text that references the their original name, e.g. First Up (formerly DVAEYC)
* I didn't change the urls/markdown titles to reflect this name change in the hopes that this change won't negatively affect SEO.


## Testing Instructions
* `git pull` 
* Navigate to http://localhost:4000/projects/. You should see this change on 
Lena Ferguson's project card in the second row.
* Click on this project card. The project page should have this text in several places.
* Navigate to http://localhost:4000/projects/shortlist/2013
* Click on First Up's row.
